doc/api/deprecations.md

```diff
@@ -3056,6 +3056,23 @@ This method was deprecated because it is not compatible with
 
 Use [`buffer.subarray`][] which does the same thing instead.
 
+### DEP0159: Deprecated `_getActiveRequestsInfo` and `_getActiveHandlesInfo`
```

Comment:
```suggestion
### DEP0159: `process._getActiveRequests()` and `process._getActiveHandles()`
```
The rest of the titles in this doc doesn't say `Deprecated` at the beginning because it's implied.

Reply:
Updated